web/controllers/respondent_controller.ex

```diff
@@ -110,41 +110,66 @@ defmodule Ask.RespondentController do
     end)
   end
 
+  def render_respondents(conn, survey_id, rows) do
+    {:ok, local_time } = Ecto.DateTime.cast :calendar.local_time()
+    {integer_survey_id, _ } = Integer.parse survey_id
+
+    entries = rows
+      |> Enum.map(fn row ->
+        %{phone_number: row, survey_id: integer_survey_id, inserted_at: local_time, updated_at: local_time}
+      end)
+
+    respondents_count = entries
+      |> Enum.chunk(1_000, 1_000, []) |> Enum.reduce(0, fn(chunked_entries, total_count)  ->
```

Comment:
Chains with more than one pipe have one pipe per line and aren't further indented.